Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -72,12 +72,15 @@ public static string OAuthToken
     public class GitHub3Plugin : GitPluginBase, IRepositoryHostPlugin
     {
         private readonly TranslationString _viewInWebSite = new("View in {0}");
-        private readonly TranslationString _tokenAlreadyExist = new("You already have an OAuth token. To get a new one, delete your old one in Plugins > Settings first.");
+        private readonly TranslationString _tokenAlreadyExist = new("You already have an personal access token. To get a new one, delete your old one in Plugins > Plugin Settings first.");
+        private readonly TranslationString _generateToken = new("Generate a GitHub personal access token");
+        private readonly TranslationString _manageToken = new("Manage GitHub personal access token");
+        private readonly TranslationString _openLinkFailed = new TranslationString("Fail to open the link. Reason: ");
```

Comment:
Should we have parameters in texts like these?

Reply:
I let you decide. Even if I think about it when writing it, I don't have a strong opinion. I though it was easier for translators (even if these strings are not translated).

Reply:
Not sure how this is handled in other languages.
Not requesting changes here, this is the common pattern now.